packages/health-gorilla-core/src/test-utils.ts

```diff
@@ -0,0 +1,5 @@
+import { expect } from 'vitest';
+
+export function expectToBeDefined<T>(value: T | undefined, message?: string): asserts value is T {
```

Comment:
extreme nitpick, but to an outside observer, this seems like more noise than utility

```
expect(x, msg).toBeDefined();
expectToBeDefined(x, msg);
```

Reply:
Yeah, very true. The difference is the type assertion which makes following lines of test code more precise.